dotCMS/src/integration-test/java/com/dotmarketing/startup/runonce/Task05370ChangeContainerPathToAbsoluteTest.java

```diff
@@ -0,0 +1,462 @@
+
+        final HashMap templateMap = (HashMap) templates.get(0);
+        assertEquals(1, templates.size());
+
+        final String drawedBody = templateMap.get("drawed_body").toString();
+        assertEquals(layout, drawedBody);
+    }
+
+    /**
+     * Method to Test: {@link Task05370ChangeContainerPathToAbsolute#executeUpgrade()}
+     * When: Exists A TemplateLayout with relative path container in the drawed_body fields and the body field is null
+     * Should: Should turn it into a Absolute Path, using the template's host
+     */
+    @Test
+    public void whenTemplateLayoutHasRelativePathButBodyIsNullShouldTurnIntoAAbsolutePath() throws IOException, DotDataException {
+
+        final String layout = String.format(jsonDrawBody, "");
+        final Host host = new SiteDataGen().nextPersisted();
+
+        checkTemplateLayout(layout);
+        final Contentlet theme = new ThemeDataGen().nextPersisted();
+        final Template template = new TemplateDataGen()
+                .title("template_test_" + System.currentTimeMillis())
+                .theme(theme)
+                .drawedBody(layout)
+                .setBodyAsNull()
+                .host(host)
+                .nextPersisted();
+
+
+        final Task05370ChangeContainerPathToAbsolute task05370ChangeContainerPathToAbsolute =
+                new Task05370ChangeContainerPathToAbsolute();
+
+        task05370ChangeContainerPathToAbsolute.executeUpgrade();
+
+        checkTemplateFromDataBase(host, template);
+    }
+
+    private void checkTemplateFromDataBase(Host host, Template template) throws DotDataException {
+        final ArrayList templates = getTemplateFromDataBase(template);
+
+        final HashMap templateMap = (HashMap) templates.get(0);
+        assertEquals(1, templates.size());
+
+        final String drawedBody = templateMap.get("drawed_body").toString();
+        assertTrue(drawedBody.contains(String.format("//%s/application/containers/default/", host.getHostname())));
+
+        if (!Strings.isNullOrEmpty(templateMap.get("body").toString())) {
+            final String body = templateMap.get("body").toString();
+            assertTrue(body.contains(String.format("//%s/application/containers/default/", host.getHostname())));
+        }
+    }
+
+    private ArrayList getTemplateFromDataBase(Template template) throws DotDataException {
+        return new DotConnect()
+                    .setSQL(String.format("select drawed_body, body from template where inode = '%s'", template.getInode()))
+                    .loadResults();
+    }
+
+    /**
+     * Method to Test: {@link Task05370ChangeContainerPathToAbsolute#executeUpgrade()}
+     * When: Exists A TemplateLayout with absolute path container
+     * Should: Should does not anything
+     */
+    @Test
+    public void whenTemplateLayoutHasAbsolutePathShouldKeepThePathWithoutChange() throws IOException, DotDataException, DotSecurityException {
```

Comment:
Codacy found an issue: [JUnit tests should include assert() or fail()](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6034685)